pyproject.toml

```
@@ -67,6 +67,7 @@ flake8-docstrings = "^1.6.0"
 pep8-naming = "^0.13.2"
 jsondiff = "^2.0.0"
 types-setuptools = "^70.3.0"
+python-semantic-release = "^9.8.5"
```

Aligning with the other repos given the pipeline config depends on that one:
```suggestion
python-semantic-release = "^7.32.2"
```